src/core/renderController.ts

```diff
@@ -91,34 +91,23 @@ export module Core {
     export function flush() {
       if (_animationRequested) {
         // Layout
-        var toCompute = d3.values(_componentsNeedingComputeLayout);
-        toCompute.forEach((c) => c.computeLayout());
+        _componentsNeedingComputeLayout.values().forEach((c: Component) => c.computeLayout());
```

Comment:
I really don't like single character variable names. We should probably move away from them